### DISCUSSION TITLE
坑爹的阿里云，才分享几下，就被封帐号了。可惜了7T空间。

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1151] [General]
阿里云盘好不容易升级到7T空间，而且其中4T是永久空间，才用ALIST分享几天，就被封帐号了，好惨！
![image](https://user-images.githubusercontent.com/61218953/169937523-48e6bf2b-1ecc-4965-885c-479d55cbd2a8.png)

![image](https://user-images.githubusercontent.com/61218953/169937564-66342c11-760a-4303-b559-22c38fd11220.png)

**Comment by anwen-anyi** [comment]
跟Alist没关系...QAQ..主要还是你使用不当QAQ...比如分享到QQ群里多IP下载/大流量下载...  建议你申诉一下就不要公开阿里云盘了 这个问题在去年就说了不知道N次了...

**Comment by laotuoyuan** [comment]
我的阿里云盘只是自己下载，因为下载了几十T，被限速了。如何才能申请解除限速？

**Reply by anwen-anyi** [reply]
如果你是用的阿里云盘下载的话直接甩工单过去,如果不是那够呛....前阵子有个人用小白羊客户端(第三方)看了2部加起来不到10G的电影被限速了提工单问怎么回事,然后巴拉巴拉巴拉一堆,最后问可以不可以解除,回复是可以解除但是要你发查询授权书和手持身份证的照片等等一堆东西..没有造谣这个就是真实发生的事情..

显然你下载了几十T用的Alist吧..... 如果你执意要解开限速的话可以参考下上面的按理你能不能接受QAQ~~~~~

**Comment by laotuoyuan** [comment]
确实用alist下载的，也发工单了，但是没有回复。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Xhofe/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 上午10:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Xhofe/alist] 坑爹的阿里云，才分享几下，就被封帐号了。可惜了7T空间。 (Discussion #1151)





 
如果你是用的阿里云盘下载的话直接甩工单过去,如果不是那够呛....前阵子有个人用小白羊客户端(第三方)看了2部加起来不到10G的电影被限速了提工单问怎么回事,要你发查询授权书和手持身份证的照片等等一堆东西..没有造谣这个就是真实发生的事情..
 
显然你下载了几十T用的Alist吧..... 如果你执意要解开限速的话可以参考下上面的按理你能不能接受QAQ~~~~~
